constance/base.py

```diff
@@ -9,6 +9,12 @@ def __init__(self):
         super(Config, self).__setattr__('_backend',
             utils.import_module_attr(settings.BACKEND)())
 
+    def __getitem__(self, key):
+        return self.__getattr__(key)
+    
+    def __setitem__(self, key, value):
+        return self.__setattr__(key, value)
+
```

Comment:
In my opinion, this variation looks simpler.
```python
    __getitem__ = __getattr__
    __setitem__ = __setattr__
```

Reply:
@bartmachielsen This must be declared after the associated methods are defined. Like that
```python
    def __getattr__(...):
        ...

    __getitem__ = __getattr__

    def __setattr__(...):
        ...

    __setitem__ = __setattr__
```